deployer/helm_upgrade_decision.py

```diff
@@ -84,26 +83,16 @@ def get_all_cluster_yaml_files(is_test=False):
     Returns:
         set[path obj]: A set of absolute paths to all cluster.yaml files in the repo
```

Comment:
this doesn't return a `set` anymore, right?

Reply:
We should do a run of `generate-helm-upgrade-jobs` locally to check as I used set operations throughout that logic and they _could_ break if this object is no longer a set. I'd hope that CI would've picked it up, but I wasn't able to write a test for everything.

Reply:
I did a local test run and `generate-helm-upgrade-jobs` was fine so this property can be edited as needed